lib/analyzer/index.ts

```diff
@@ -16,12 +18,26 @@ const extractActions = [
   ...aptAnalyzer.APT_PKGPATHS,
   ...apkAnalyzer.APK_PKGPATHS,
   ...osReleaseDetector.OS_VERPATHS,
-].map((p) => {
-  return {
-    name: "txt",
-    pattern: p,
-  };
-});
+]
+  .map((p) => {
+    return {
+      name: "txt",
+      pattern: p,
+    };
+  })
+  .concat(
+    [...hashAnalyzer.HASH_PKGPATHS].map((p) => {
```

Comment:
I think it will be better to create 2 different consts (one for every mapping) and than combine them together